cartesian_impedance_msgs/msg/SetCartesianImpedance.msg

```diff
@@ -1,10 +1,13 @@
 # Here we combine all the parameters message definitions into a single message
 # that needs to be send to the controller in order to adjust the Cartesian Impedance Parameters
 
-string               tool
-CartesianStiffness   stiffness
-CartesianDamping     damping
-MaxCartesianVelocity max_cart_vel
-MaxControlForce      max_ctrl_force
-MaxPathDeviation     max_path_deviation
-NullSpace            null_space_params
+string                 tool
+CartesianStiffness     stiffness
+#use this if the damping is in Ns/m, Ns/rad
```

Comment:
If I may make a suggestion, I think the choice for either ratio or Ns/m is something we should get as much input on as possible.

I've talked to multiple people here (control engineers, mechanical engineers, ..) and have heard passionate arguments both for and against both options. Some find ratios much more intuitive, others dislike the relativeness of them. Ns/m is regarded as difficult, and not as 'platform independent' (ie: 0.5 is always 0.5, but the effect of 1.0 Ns/m isn't immediately clear), even though it is expressed with actual an unit (so it is independent in a way).

Personally I agree with @shaun-edwards (and I think I've said it before), in that any message should be unambiguously interpretable. Making the semantics of fields dependent on implicit or invisible assumptions / knowledge seems like a bad idea.